apps/backend/src/libs/modules/plugins/libs/constants/open-routes.ts

```diff
@@ -0,0 +1,6 @@
+const OPEN_ROUTES: Set<RegExp> = new Set([
```

Comment:
Can't we use `as const` and usual array instead of Set?
```suggestion
const OPEN_ROUTES: Set<RegExp> = new Set([
```